### DISCUSSION TITLE
Documentation for DI integration

### DISCUSSION THREAD

**Discussion by lol768** [quartznet/quartznet#979] [General] [answered]
I'm reading https://www.quartz-scheduler.net/documentation/quartz-3.x/packages/microsoft-di-integration.html#di-aware-job-factories

```csharp
// quickest way to create a job with single trigger is to use ScheduleJob (requires version 3.2)
```

Method doesn't exist in latest NuGet release? Is 3.2 available?

```chsarp
        // here's a known job for triggers
        var jobKey = new JobKey("awesome job", "awesome group");
        q.AddJob<ExampleJob>(jobKey, j => j
            .WithDescription("my awesome job")
        );
```

There is no such overload for AddJob that accepts a `JobKey` that I can see (though there is one in the code https://github.com/quartznet/quartznet/blob/b8fe74596b096b1803654c66110989e38114e0c5/src/Quartz.Extensions.DependencyInjection/ServiceCollectionExtensions.cs#L92-L109).. Looks like I need to use `WithIdentity` and have the `JobKey` instance created for me?

Where are the docs for the current _stable_ release, as available on NuGet?

**Comment by lahma** [comment, marked as answer]
This is because I try to keep up with the docs while I develop/tune new features. Release 3.2 will happen soon'ish so the documentation already has it. 

Yes for now you need to use the `WithIdentity`, I'm trying to make the API more idiomatic for use cases in 3.2 as the 3.1 was first introduction to it for collecting feedback (please give a shout if there's more that you need!). 

If you already want the get 3.2 preview bits, you just need to point your NuGet to [the preview feed](https://www.myget.org/F/quartznet/api/v3/index.json).

**Reply by lahma** [reply]
The 3.2 release is now available.

**Reply by lol768** [reply]
Thanks @lahma!
